src/storage/src/hummock/iterator/reverse_user.rs

```diff
@@ -49,7 +49,7 @@ pub struct ReverseUserIterator {
     read_epoch: Epoch,
 
     /// Ensures the SSTs needed by `iterator` won't be vacuumed.
-    _version: Option<Arc<ScopedLocalVersion>>,
+    _version: Option<Arc<PinnedVersion>>,
```

Comment:
If we can ensure that executor will only use `Hummock::scan` rather than `Hummock::iter`, we only need to hold `Arc<PinnedVersion>` in the beginning of `scan`.

Reply:
Emmm. It seems not work. In some case `iter` is necessary.